src/main/java/edu/harvard/iq/dataverse/api/AbstractApiBean.java

```diff
@@ -601,7 +601,7 @@ protected DvObject findDvo(@NotNull final String id) throws WrappedResponse {
      * @throws WrappedResponse
      */
     @NotNull
-    protected DvObject findDvoByIdAndFeaturedItemTypeOrDie(@NotNull final String dvIdtf, String type) throws WrappedResponse {
+    protected DvObject findDvoByIdAndTypeOrDie(@NotNull final String dvIdtf, String type, Boolean testForReleased) throws WrappedResponse {
```

Comment:
`testForReleased` could be a primitive, simply `boolean`.

Also the name `testForReleased` isn't very clear. If what it does is enable the validation for featured items, why did you name it that way?